src/Burgomaster.php

```diff
@@ -44,7 +44,7 @@ public function __construct($stageDir, $projectRoot = null)
 
         $this->debug("Creating staging directory: $this->stageDir");
 
-        if (!mkdir($this->stageDir)) {
+        if (!mkdir($this->stageDir, 0777, true)) {
```

Comment:
There's no real reason for it to be this open. Group writable should be sufficient.